rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/csharp/CSharp50Generator.kt

```diff
@@ -245,6 +245,18 @@ open class CSharp50Generator(
         is Member.Method -> publicName
     }
 
+    protected open fun Member.ctorSubstitutedName(scope: Declaration) = when (this) {
```

Comment:
Let's call this `creationExpressionSubstituted`? To avoid confusion between ctor name (without `new`) and this.